### PR TITLE
[RO-357] Phpseclib security vulnerability fix

### PR DESCRIPTION
**Pull request by Skyld**
Ticket: https://oat-sa.atlassian.net/browse/RO-357

Fix for: https://github.com/advisories/GHSA-vf4w-fg7r-5v94

Had to upgrade https://github.com/oat-sa/bundle-lti1p3 from `v3.x` to `v4.x` in order to fix `phpseclib/phpseclib` security issue.

### CODE REVIEW

**Review by luisfn — APPROVED** [review]
**Review Checklist**
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful
- [X] Changelog is updated according to changes (if applicable)
- [X] Documentation is updated according to changes (if applicable)

**Review by luisfn — APPROVED** [review]

File: src/Security/Generator/JwtTokenGenerator.php [hunk 1]
```diff
@@ -71,11 +72,11 @@ public function create(UserInterface $user, Request $request, string $subjectCla
             // jti claim
             ->identifiedBy($this->uuidFactory->uuid4()->toString())
             // iat claim
-            ->issuedAt($currentTime)
+            ->issuedAt((new DateTimeImmutable())->setTimestamp($currentTime))
             // nbf claim
-            ->canOnlyBeUsedAfter($currentTime)
+            ->canOnlyBeUsedAfter((new DateTimeImmutable())->setTimestamp($currentTime))
```
Comment (uncleempty) [inline src/Security/Generator/JwtTokenGenerator.php:77]: Not that it reall matters but could you extract `(new DateTimeImmutable())->setTimestamp($currentTime)` from this line and line 75 into additional variable, please?

**Review by uncleempty — CHANGES_REQUESTED** [review]
Almost there, just a simple comment to discuss :) 
Good job! :+1:

**Review by uncleempty — APPROVED** [review]
**Review Checklist**
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful
- [X] Changelog is updated according to changes (if applicable)
- [X] Documentation is updated according to changes (if applicable)

**Review by luisfn — APPROVED** [review]

**Pull request merged**